google/cloud/aiplatform/docker_utils/build.py

```diff
@@ -275,14 +264,18 @@ def _get_relative_path_to_workdir(
         The relative path to the workdir or None if path is None.
 
     Raises:
-        ValueError: If the path is not relative to the workdir.
+        ValueError: If the path does not exist or is not relative to the workdir.
     """
     if path is None:
         return None
 
+    if not Path(path).is_file():
+        raise ValueError(f'The {value_name} "{path}" must exist.')
     if not path_utils._is_relative_to(path, workdir):
         raise ValueError(f'The {value_name} "{path}" must be in "{workdir}".')
-    return Path(path).relative_to(workdir).as_posix()
+    abs_path = Path(path).expanduser().resolve()
```

Comment:
Why is this change needed?

Reply:
I found there will be an issue if users use absolute paths and relative paths interchangeably. Therefore, use the absolute paths always.